src/types/plugin-input.ts

```diff
@@ -50,4 +50,4 @@ export const pluginSettingsSchema = T.Object(
 export type PluginSettings = StaticDecode<typeof pluginSettingsSchema>;
 
 export type SupportedEvents = "issues.closed";
-export type ContextPlugin = Context<PluginSettings, EnvConfig, SupportedEvents>;
+export type ContextPlugin = Context<PluginSettings, EnvConfig, null, SupportedEvents>;
```

Comment:
Looks strange to have `null` in the template?

Reply:
What do you suggest? I could make it optional by putting it as last

Reply:
Yes just realized it is for the `TCommand`. Shouldn't the SDK be released first and updated in the package.json for this to work? Otherwise it won't compile.

Reply:
Yes, can you publish the SDK?

Reply:
I will. All it takes is a merge on main so you should be able to do so as well